pkg/account/api/admin_account.go

```diff
@@ -412,6 +417,12 @@ func (s *AccountService) getMyOrganizations(
 		if accWithOrg.AccountV2.Disabled || accWithOrg.Organization.Disabled || accWithOrg.Organization.Archived {
 			continue
 		}
+		// If the account is an admin account, we append the organization.
+		// Otherwise, we check if the account is enabled in any environment in this organization.
+		if accWithOrg.AccountV2.OrganizationRole == accountproto.AccountV2_Role_Organization_ADMIN {
+			myOrgs = append(myOrgs, accWithOrg.Organization)
+			continue
+		}
```

Comment:
Because the admin account doesn't have environment roles, we must append the organization.